docs/releases/1.11.txt

```diff
@@ -483,6 +483,10 @@ Miscellaneous
   ``migrate`` command. Use the new :djadmin:`remove_stale_contenttypes` command
   instead.
 
+* ETags are now parsed according to the updated RFC 7232 Conditional Requests
+  specification. The ETag syntax there is slightly different from that of the
+  older RFC 2616 specification which was used in previous versions.
```

Comment:
Is it worth giving an example of how this might affect a project or will it be obvious to ETag users?